src/modules/dashboard/components/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx

```diff
@@ -404,12 +404,16 @@ const CreatePaymentDialogForm = ({
             placeholder={MSG.userPickerPlaceholder}
           />
         </div>
-        {values.recipient && isConfusing(values.recipient.profile.username) && (
-          <ConfusableWarning
-            walletAddress={values.recipient.profile.walletAddress}
-            colonyAddress={colonyAddress}
-          />
-        )}
+        {values.recipient &&
+          isConfusing(
+            values.recipient.profile.username ||
+              values.recipient.profile.displayName,
```

Comment:
Quick nitpicky question, should this be part of this PR?

Reply:
It's not included in the issue no, this is a fix for an unrelated bug I found while working on the issue